docker-compose error Module not found: Error: Can't resolve './pages/DashBoard' in '/view/src' in local testing

Hello @JosueDavalos I wonder if you can quickly help me 😃 

I would need to add a small thing as in #11 but when I try to build locally (without any modifications) I get

```
 => [build-stage 2/7] WORKDIR /view                                                                                0.2s
 => [build-stage 3/7] COPY package.json .                                                                          0.0s
 => [build-stage 4/7] RUN npm install                                                                             85.8s
 => [build-stage 5/7] RUN npm audit fix                                                                            6.3s
 => [build-stage 6/7] COPY . .                                                                                     0.0s
 => ERROR [build-stage 7/7] RUN npm run build                                                                     20.9s
------
 > [build-stage 7/7] RUN npm run build:
#13 0.585
#13 0.585 > view@1.0.0 build /view
#13 0.585 > webpack --mode production
#13 0.585
#13 20.79 assets by status 734 KiB [cached] 5 assets
#13 20.79 Entrypoint main = src/main.css bundle.js
#13 20.79 orphan modules 14.6 MiB (javascript) 5.49 KiB (runtime) [orphan] 384 modules
#13 20.79 runtime modules 884 bytes 4 modules
#13 20.79 cacheable modules 14.6 MiB (javascript) 173 KiB (css/mini-extract)
#13 20.79   javascript modules 14.6 MiB 45 modules
#13 20.79   css modules 173 KiB
#13 20.79     modules by path ./src/ 4.77 KiB
#13 20.79       css ./node_modules/css-loader/dist/cjs.js!./src/index.css 589 bytes [built] [code generated]
#13 20.79       css ./node_modules/css-loader/dist/cjs.js!./src/components/home/Home.css 4.08 KiB [built] [code generated]
#13 20.79       css ./node_modules/css-loader/dist/cjs.js!./src/components/footer/Footer.css 61 bytes [built] [code generated]
#13 20.79       css ./node_modules/css-loader/dist/cjs.js!./src/components/manage/ReportProblem.css 60 bytes [built] [code generated]
#13 20.79     modules by path ./node_modules/ 168 KiB
#13 20.79       css ./node_modules/css-loader/dist/cjs.js!./node_modules/bootstrap/dist/css/bootstrap.min.css 158 KiB [built] [code generated]
#13 20.79       css ./node_modules/css-loader/dist/cjs.js!./node_modules/react-toastify/dist/ReactToastify.css 10 KiB [built] [code generated]
#13 20.79
#13 20.79 ERROR in ./src/App.jsx 8:0-42
#13 20.79 Module not found: Error: Can't resolve './pages/DashBoard' in '/view/src'
#13 20.79 resolve './pages/DashBoard' in '/view/src'
#13 20.79   using description file: /view/package.json (relative path: ./src)
#13 20.79     Field 'browser' doesn't contain a valid alias configuration
#13 20.79     using description file: /view/package.json (relative path: ./src/pages/DashBoard)
#13 20.79       no extension
#13 20.79         Field 'browser' doesn't contain a valid alias configuration
#13 20.79         /view/src/pages/DashBoard doesn't exist
#13 20.79       .js
#13 20.79         Field 'browser' doesn't contain a valid alias configuration
#13 20.79         /view/src/pages/DashBoard.js doesn't exist
#13 20.79       .jsx
#13 20.79         Field 'browser' doesn't contain a valid alias configuration
#13 20.79         /view/src/pages/DashBoard.jsx doesn't exist
#13 20.79       as directory
#13 20.79         /view/src/pages/DashBoard doesn't exist
#13 20.79  @ ./src/index.js 5:0-24 6:107-110
#13 20.79
#13 20.79 webpack 5.64.4 compiled with 1 error in 19556 ms
#13 20.84 npm ERR! code ELIFECYCLE
#13 20.84 npm ERR! errno 1
#13 20.85 npm ERR! view@1.0.0 build: `webpack --mode production`
#13 20.85 npm ERR! Exit status 1
#13 20.85 npm ERR!
#13 20.85 npm ERR! Failed at the view@1.0.0 build script.
#13 20.85 npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```

Do you have an idea why this happens?

I've done nothing else than

```
git clone git@github.com:cms-dpoa/CMS_Trivia.git
cd CMS_Trivia/
docker-compose up --build
```

FYI: I had to reinstall my linux distribution, but I guess it should not matter, it should install all that is needed, and I do not see any errors in the `npm install` step.

--- a/view/src/App.jsx
+++ b/view/src/App.jsx
@@ -5,7 +5,7 @@ import GameSection from "./pages/GameSection";
 import NotFound from "./pages/NotFound";
 import Home from "./pages/Home";
 import MyScore from "./pages/MyScore";
-import DashBoard from "./pages/DashBoard";
+import Dashboard from "./pages/Dashboard";
 import Layout from "./components/Layout";
 import generateStore from "./redux/store";
 import MillionaireGame from "./components/game/MillionaireGame";
@@ -24,7 +24,7 @@ function App() {
             <Route exact path="/play" component={GameSection} />
             <Route exact path="/play/millionaire" component={MillionaireGame} />
             <Route exact path="/myscore" component={MyScore} />
-            <Route exact path="/dashboard" component={DashBoard} />
+            <Route exact path="/dashboard" component={Dashboard} />
             <Route exact path="/manage" component={Manage} />
             <Route component={NotFound} />
           </Switch>

